modules/http/src/main/resources/META-INF/mule-httpn.xsd

```diff
@@ -117,7 +117,16 @@
             <xsd:extension base="mule:abstractExtensionType">
                 <xsd:sequence>
                     <xsd:element ref="tls:context" minOccurs="0" maxOccurs="1" />
-                    <xsd:element name="worker-threading-profile" type="mule:asynchronousThreadingProfileType" minOccurs="0" maxOccurs="1"/>
+                    <xsd:choice>
+                        <xsd:element name="worker-threading-profile" type="mule:asynchronousThreadingProfileType" minOccurs="0" maxOccurs="1"/>
+                        <xsd:element name="direct-threading-profile" minOccurs="0" maxOccurs="1">
+                            <xsd:annotation>
+                                <xsd:documentation>
+                                    Use this threading profile to disable the use of a worker thread pool for processing requests and instead process requests using the selector threads.
```

Comment:
Do you mean "for processing requests and instead process requests"?  This is fine.